src/makeCalculator.js

```diff
@@ -37,7 +37,51 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    operate(callback, number) {
+      this.result = callback(number);
+
+      return this;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(number) {
+      let result = calculator.result;
+
+      result += number;
+
+      return result;
```

Comment:
```suggestion
      return calculator.result;
```

Reply:
don't return some computed value, use the field "result" of your calculator object

Reply:
fixed